workspace-manager-clienttests/src/main/resources/suites/BasicIntegration.json

```diff
@@ -8,7 +8,8 @@
     "integration/CreateGetDeleteWorkspace.json",
     "integration/DataReferenceLifecycle.json",
     "integration/EnumerateDataReferences.json",
-    "integration/GetRoles.json"
+    "integration/GetRoles.json",
+    "integration/CreateGetDeleteControlledGcsBucket.json"
```

Comment:
should any of these run as perf tests? I'm not exactly sure what our test strategy is there

Reply:
We have not really worried performance yet. I don't think we have performance targets specified.
My focus is to make sure we have robust integration tests and worry performance later.